### ISSUE TITLE
Bug: current content pinned not right

### ISSUE THREAD

**Issue by dqnykamp** [Doenet/DoenetTools#1640]
The below image shows the current content on Sept 20.  The Beginning of semester survey was assigned on Sept 6, but it looks like it was assigned this week.  Also, it set to be pinned until Sept 16, but it is still showing up as pinned.

![image](https://user-images.githubusercontent.com/1037056/191381898-22251f5d-9a69-4ab4-8296-943d0f431f2c.png)

![image](https://user-images.githubusercontent.com/1037056/191382222-2eb63f77-f9a6-4bc4-bfb2-be8f4a6d22bb.png)

**Comment by dqnykamp** [comment]
Strange how this survey is still pinned, but the Practice Gateway Exam dropped off the pinned list like it was supposed to this morning.
![image](https://user-images.githubusercontent.com/1037056/191890036-8ca3f2d1-0123-467f-863e-5da76d1b945e.png)

**Comment by dqnykamp** [comment]
I finally figured out why the Beginning of semester survey will not drop off the pinned list, while the others did.  The Beginning of semester survey also has a due date, so it is always returned by loadTODO.php.  The other pinned items stop being loaded by loadTODO.php when their pinned date is done since the pin was the only reason they were retrieved.

loadTODO.php does not even return pinnedUntilDate, so there is no way to know to stop pinning that item.

Basically pinning something that is also assigned is broken.
